tests/mutation_test.cc

```diff
@@ -1320,6 +1320,99 @@ SEASTAR_THREAD_TEST_CASE(test_mutation_upgrade_type_change) {
     assert_that(m).is_equal_to(m2);
 }
 
+SEASTAR_THREAD_TEST_CASE(test_row_marker) {
+    auto ttl = std::chrono::seconds(1);
```

Comment:
nitpick: in C++, there is no need to define variables at the top of the function. It's clearer if you define them right before actually using them. By the time you used "ttl" below, I forgot what it was :-)
By the way, you can use "const" to emphasize you only set it once, and don't plan to change it below.

Reply:
Added `const` qualifiers and increased locality of the code.

Reply:
You can also write `auto ttl = 1s;` if you want.